official/vision/beta/configs/multitask_config.py

```diff
@@ -0,0 +1,267 @@
+  model_output_keys) removed."""
+  model: SemanticSegmentationModelSpecs = SemanticSegmentationModelSpecs()
+  train_data: SegmentationDataConfig = SegmentationDataConfig(is_training=True)
+  validation_data: SegmentationDataConfig = SegmentationDataConfig(is_training=False)
+  losses: SegmentationLosses = SegmentationLosses()
+  evaluation: SegmentationEvaluation = SegmentationEvaluation()
+  train_input_partition_dims: List[int] = dataclasses.field(
+      default_factory=list)
+  eval_input_partition_dims: List[int] = dataclasses.field(
+      default_factory=list)
+
+
+@exp_factory.register_config_factory('multitask_vision')
+def multitask_vision() -> multi_cfg.MultiTaskExperimentConfig:
+  """
+  Vision task with single backbone and multiple heads.
+  Each head can be a segmenter, detector or classifier.
+  TODO: support n heads instead of just one segmenter, one classifier.
+  TODO: use same num_class and input_size in both task and model definition
+  TODO: check where eval steps is used
+
+  multi_cfg.MultiTaskConfig:
+    - Retains each task_name, entire task, eval_steps and weights,
+        - Entire_task used in respective multitask trainers for train_step
+        - Weights used in task_sampler
+  
+  multi_cfg.MultiTaskTrainerConfig:
+    - trainer_type and task_sampler used to configure task sampling in train_lib
+    - Normal multi_cfg.TrainerConfig params used directly in train_lib
+  """
+  input_path_segmentation = ''
+  input_path_classification = ''
+  steps_per_epoch = 6915
+  train_batch_size = 1
+  eval_batch_size = 1
+  validation_steps = 6915
+
+  segmentation_routine = multi_cfg.TaskRoutine(
+    task_name='segmentation',
+    task_config=SemanticSegmentationSubtask(
+      model=SemanticSegmentationModelSpecs(
+          num_classes=19,
+          input_size=[256, 256, 3]),
+      losses=SegmentationLosses(
+          ignore_label=250,
+          top_k_percent_pixels=0.3),
+      train_data=SegmentationDataConfig(
+          output_size=[256, 256],
+          input_path=input_path_segmentation,
+          global_batch_size=train_batch_size,
+          is_training=True,
+          aug_scale_min=0.5,
+          aug_scale_max=2.0,
+          preserve_aspect_ratio=False,
+          aug_policy='randaug',
+          randaug_magnitude=5,
+          randaug_available_ops=[
+            'AutoContrast', 'Equalize', 'Invert', 'Rotate', 'Posterize', 
+            'Solarize', 'Color', 'Contrast', 'Brightness', 'Sharpness', 
+            'Cutout', 'SolarizeAdd']),
+      validation_data=SegmentationDataConfig(
+          output_size=[256, 256],
+          input_path=input_path_segmentation,
+          global_batch_size=eval_batch_size,
+          is_training=False,
+          resize_eval_groundtruth=True,
+          drop_remainder=False)),
+    eval_steps=None, # check where eval steps is used
+    task_weight=1.0
+  )
+  classification_routine = multi_cfg.TaskRoutine(
```

Comment:
just wondering, should this be detector routine? Or are you doing a classifier?

Reply:
It is a classifier that predicts labels per image. 
It does not predict location (bounding boxes) of objects yet, but is easily extended to do so.